app/models/user.rb

```diff
@@ -2,13 +2,13 @@ class User < ApplicationRecord
   validates :name, presence: true
 
   include Humanizer
-  before_save :get_gravatar_url
+  before_save :the_gravatar_url
```

Comment:
Bu değişikliğin amacı nedir? `set_gravatar_url` gibi bir şey yapmak daha açıklayıcı olur bence, `the_gravatar_url` o fonksiyonla ilgili hiçbir şey anlatmıyor çünkü

Reply:
rubocop ikazından ötürü öyle bir değişiklik yaptım. açıklaması da şurada: https://github.com/bbatsov/ruby-style-guide#accessor_mutator_method_names